recipe/build.sh

```diff
@@ -38,19 +36,34 @@ fi
 
 INSTALL_PREFIX=${PREFIX}/lib/clang/${PKG_VERSION}
 
+# make sure we use our pre-built libcxx, see
+# https://github.com/llvm/llvm-project/blame/llvmorg-14.0.0/compiler-rt/CMakeLists.txt#L178-L181
+export CMAKE_CXX_FLAGS="$CMAKE_CXX_FLAGS -stdlib=libcxx"
+# the following option is confusingly named; it means not rebuild libcxx, see
+# https://github.com/llvm/llvm-project/blame/llvmorg-14.0.0/compiler-rt/CMakeLists.txt#L607-L608
+CMAKE_ARGS="$CMAKE_ARGS -DCOMPILER_RT_USE_LIBCXX=OFF"
+
+# point compiler-rt to correct C++ stdlib
+if [[ "$target_platform" == osx-* ]]; then
+    CMAKE_ARGS="$CMAKE_ARGS -DCOMPILER_RT_HAS_LIBCXX=1"
+else
+    CMAKE_ARGS="$CMAKE_ARGS -DCOMPILER_RT_HAS_LIBSTDCXX=1"
+fi
+
 cmake \
     -G "Unix Makefiles" \
     -DCMAKE_BUILD_TYPE="Release" \
     -DLLVM_CONFIG_PATH="$PREFIX/bin/llvm-config" \
+    -DLLVM_EXTERNAL_LIT="PREFIX/bin/lit" \
```

Comment:
PREFIX -> $PREFIX

Reply:
Duh, not sure how this one slipped through... PR incoming

Reply:
#45